build/main.rs

```diff
@@ -120,24 +120,9 @@ cbindgen::generate(&manifest_dir).unwrap().write_to_file("path/to/file.h")
         format_generated_file(static_atoms_path.as_path());
     }
 
-    // Generate C bindings with cbindgen if it is available
-    if has_cbindgen {
-        generate_c_bindings();
-    }
-
     println!("cargo:rerun-if-changed=src/");
 }
 
-// Function to generate C bindings using cbindgen
-fn generate_c_bindings() {
-    let bindings = generate_with_config("./", cbindgen::Config::from_file("./.cbindgen.toml").unwrap())
-        .expect("Unable to generate bindings");
-
-    // Write bindings to the docs/shared_library/libscryer_prolog.h file.
-    let out_path = PathBuf::from("./docs/shared_library/");
-    bindings.write_to_file(out_path.join("libscryer_prolog.h"));
-}
-
```

Comment:
Oh, I completely overlooked that you were already using cbindgen as a library here.
So just the check for whether cbindgen is installed was unnecessary.
I think having it outlined into its own function was actually better.